src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1490,6 +1490,82 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   </Target>
 
   <!--
+    ====================================================================================
+                                        _GetProjectReferenceTargetFrameworkProperties
+
+    Builds the GetTargetFrameworkProperties target of all existent project references,
+    passing $(TargetFrameworkMoniker) as $(ReferringTargetFramework) and sets the
+    SetTargetFramework metadata of the project reference to the value that is returned.
+
+    This allows a cross-targeting project to select how it should be configured to
+    build against the most appropriate target for the referring target framework.
+
+    ======================================================================================
+  -->
+  <Target Name="_GetProjectReferenceTargetFrameworkProperties" Outputs="%(_MSBuildProjectReferenceExistent.Identity)">
```

Comment:
I'm having trouble getting things to work with this change. :( Given the time constraint (we want this in today so that the new (hopefully-not-but-potentially-breaking) targets are in Preview 5. Would you be ok iterating on the final pattern here as an immediate follow-up change?